Automattic/wp-calypso
# Color Schemes: Add postcss custom properties to main style build step

#### This PR 
- is part of a series of PRs aiming to provide support for Color Schemes in Calypso.

-  focuses on adding the postcss plugin `postcss-custom-properties` to the main style build step

-  adds `postcss.config.json` as a central place for postcss plugin configurations

- is a continuation of PR https://github.com/Automattic/wp-calypso/pull/17366, which will be closed in favor of this and a few other smaller PRs.

#### How to test

Run the CSS build and ensure that the output is identical to the output generated prior to this PR.

#### Dependencies (🚨)

This PR depends on functionality added in PR https://github.com/Automattic/wp-calypso/pull/18905. That PR has to be merged to master prior to this one.
